src/api/api.tsx

```diff
@@ -98,6 +98,8 @@ export default {
   deleteEvent: (eventId: Event['id']) => IFetch<RequestResponse>({ method: 'DELETE', url: `${EVENTS_ENDPOINT}/${String(eventId)}/` }),
   notifyEventRegistrations: (eventId: Event['id'], title: string, message: string) =>
     IFetch<RequestResponse>({ method: 'POST', url: `${EVENTS_ENDPOINT}/${String(eventId)}/notify/`, data: { title, message } }),
+  sendFilesToAttendees: (eventId: Event['id'], files: File | File[] | Blob) =>
```

Comment:
Foreslår at vi endrer til `sendGiftCardsToAttendees` siden endepunktet og innholder er spesifikt laget for det